docs/source/about_metrics.rst

```diff
@@ -0,0 +1,22 @@
+All about metrics
+=================
+
+🤗 Datasets provides access to a wide range of NLP metrics. You can load metrics associated with benchmark datasets like GLUE or SQuAD, and complex metrics like BLEURT or BERTScore, with a single command: :func:`datasets.load_metric`. Once you've loaded a metric, easily compute and evaluate a model's performance.
+
+ELI5: ``load_metric``
+-------------------------------------------
+
+Loading a dataset and loading a metric share many similiarites. This was an intentional design choice because we wanted to create a simple and unified experience. When you call :func:`datasets.load_metric`, the metric loading script is downloaded and imported from Github (if it hasn't already been downloaded before). It contains information about the metric such as it's citation, homepage, and description.
```

Comment:
```suggestion
Loading a dataset and loading a metric share many similiarites. This was an intentional design choice because we wanted to create a simple and unified experience. When you call :func:`datasets.load_metric`, the metric loading script is downloaded and imported from GitHub (if it hasn't already been downloaded before). It contains information about the metric such as it's citation, homepage, and description.
```